### ISSUE TITLE
[DataCap Application]Toastmasters

### ISSUE THREAD

**Issue by Jacintos** [filecoin-project/filecoin-plus-large-datasets#162]
# Large Dataset Notary Application

To apply for DataCap to onboard your dataset to Filecoin, please fill out the following.

## Core Information
- Organization Name: Toastmasters International
- Website / Social Media: https://www.toastmasters.org/
- Total amount of DataCap being requested (between 500 TiB and 5 PiB): 5PiB
- Weekly allocation of DataCap requested (usually between 1-100TiB): 300TiB
- On-chain address for first allocation: f17okow3rxzyym7zjt3sxdx7z2fubguczn34kz3qq

_Please respond to the questions below by replacing the text saying "Please answer here". Include as much detail as you can in your answer._

## Project details

Share a brief history of your project and organization.
```
Toastmasters began as a series of speaking clubs organized by Ralph C. Smedley during his time working for the YMCA (Young Men's Christian Association) in Bloomington, Illinois, United States. As director of education at the YMCA, Smedley saw a need for the men in the community to learn how to speak, conduct meetings, plan programs and work on committees, and he wanted to help them. Smedley decided to organize a club where they could learn these skills in a social environment, and the men responded well to the concept. He named the group the Toastmasters Club; "toastmaster" was a popular term that referred to a person who gave toasts at banquets and other occasions.
The first unofficial Toastmasters meeting was held on March 24, 1905. Much like Toastmasters meetings today, the participants took turns leading and speaking at each meeting. Once the new YMCA building was christened in April of 1924, Smedley was able to introduce the idea of Toastmasters to his colleagues, and organize the Toastmasters club that eventually became Club No. 1 of Toastmasters International. The first meeting was held at the YMCA building on October 22, 1924. Word spread about Smedley's Toastmasters clubs and soon people in surrounding communities and other states were asking how they could start their own club.
To save time replying to the many letters and inquiries he was receiving, Smedley wrote the "Manual of Instructions" and "Ten Lessons in Public Speaking," which he had printed and bound in paper covers. On October 25, 1928, he secured copyrights on the publications and trademarked the name "Toastmasters Club."
```

What is the primary source of funding for this project?
```
New member fees and some of the most famous names in business have sponsored clubs.
```

What other projects/ecosystem stakeholders is this project associated with?
```
No.
```

## Use-case details

Describe the data being stored onto Filecoin
```
Toastmasters International teaches public speaking and leadership skills through a worldwide network of clubs. A large amount of teaching videoes need to be stored and showed in public.
```

Where was the data in this dataset sourced from?
```
It's from the official website where these videoes are showed mainly. These were uploaded by employees.
```

Can you share a sample of the data? A link to a file, an image, a table, etc., are good ways to do this. 
```
Sure.
[Speechcrafter Overview for Coordinators.mp4.zip](https://github.com/filecoin-project/filecoin-plus-large-datasets/files/7761826/Speechcrafter.Overview.for.Coordinators.mp4.zip)
```
        
Confirm that this is a public dataset that can be retrieved by anyone on the Network (i.e., no specific permissions or access rights are required to view the data).
```
Anyone who are interested in our theme or want to become more effective communicators and leaders can refer to it.
```

What is the expected retrieval frequency for this data?
```
For the time being it will not very frequent. About 3 times a year. And it will be decided according to the actual situation.
```

For how long do you plan to keep this dataset stored on Filecoin?
```
2 years. Maybe extend it in the future.
```

## DataCap allocation plan

In which geographies (countries, regions) do you plan on making storage deals?
```
Asia and North America.
```

How will you be distributing your data to storage providers? Is there an offline data transfer process?
```
We will try both online transfer process and offline transfer process. It depends on our network situation.
```

How do you plan on choosing the storage providers with whom you will be making deals? This should include a plan to ensure the data is retrievable in the future both by you and others.
```
Miners‘ with Miners enough experience are the first choice. And their price and location will also be taken into consideration.
```

How will you be distributing deals across storage providers?
```
It will be allocated according to their ability.
```

Do you have the resources/funding to start making deals as soon as you receive DataCap? What support from the community would help you onboard onto Filecoin?
```
Yes. We are ready.
```

**Comment by galen-mcandrew** [comment]
This looks like a resubmission of a previous issue: https://github.com/filecoin-project/filecoin-plus-large-datasets/issues/128

That issue was already approved by root key holders and a multisig notary was created. What happened to the previous GitHub account and issue? Please provide some more details before we take further steps on creating another notary.

**Comment by Jacintos** [comment]
Oh it can be a killer! I can not find that issue several days ago. It was vanised! I have asked for help from official but I've got no reply. Then I made a new one like this.

**Comment by Jacintos** [comment]
Which details are you want to see? My previous email pages can show you by email.

**Comment by Jacintos** [comment]
Are there any details you want here?

**Comment by TimWilliams00** [comment]
He has communicated with me, and indeed that issue disappeared. It seems to be due to the account and system. I can prove it.

**Comment by XnMatrixSV** [comment]
Happy to support. I've used Toastmasters.

**Comment by Jacintos** [comment]
Thank you guys! @TimWilliams00 @XnMatrixSV

**Comment by Jacintos** [comment]
Will my issue go on? It has started several days ago! @galen-mcandrew @dkkapur @KwrayFF

**Comment by dkkapur** [comment]
@Jacintos - yes, just need to figure out what happened to your previous application for audit trail purposes. What happened to the issue and the account? Did you get any notification from GitHub? 

@TimWilliams00 do you happen to happen any additional info on screenshots you can share as well?

**Comment by Jacintos** [comment]
My account was suddenly signed as flagged with no reason.
![16F71030-2E07-47C0-9516-A0FC901897BE](https://user-images.githubusercontent.com/93566395/147850552-90fa7598-e75b-4793-a475-122b8f8662cc.jpeg)

I had contacted support but got no reply. So I have to make a new issue like this to apply for datacap.

**Comment by Jacintos** [comment]
My previous account was suddenly signed as flagged with no reason.
![71E82A2B-C4DB-42AE-B141-94EAD847022C](https://user-images.githubusercontent.com/93566395/147850578-7e195a9a-0eda-4aba-87c6-49c7cff27d3a.jpeg)

I had contacted support but got no reply. So I have to make a new issue like this to apply for datacap.

**Comment by TimWilliams00** [comment]
Yes, I suggest the client resubmit a new issue before the RKH signature can realize the migration. At present, the instability of GitHub system has affected the progress of LDN, but we really don't have a better solution. Last month I invited the chairman of Toastmaster Club to understand and try to use filecoin to store.

**Comment by Jacintos** [comment]
Hi Deep @dkkapur  and Galen @galen-mcandrew , sorry to bother your Holiday, we were waiting for your response for 4 days. Any update on my application?

**Comment by galen-mcandrew** [comment]
## Multisig Notary Reconnection Request
#### Multisig Notary Address
> f01605411
#### Client Address
> f17okow3rxzyym7zjt3sxdx7z2fubguczn34kz3qq
#### Notary Governance Issue
> https://github.com/filecoin-project/notary-governance/issues/344

**Comment by galen-mcandrew** [comment]
@fabriziogianni7 does the above look correct? I don't know that we've shipped those changes, but we could test it here.

**Comment by Jacintos** [comment]
@galen-mcandrew @fabriziogianni7 Please help us solve the problem, thank you!!!!!!!

**Comment by fabriziogianni7** [comment]
@galen-mcandrew this feature is still in test env

**Comment by Jacintos** [comment]
Ah how can I deal with the problem? @fabriziogianni7

**Comment by galen-mcandrew** [comment]
@Jacintos we are still testing this to recover the existing multisig and get it connected to your new application. You can make a direct request to a notary, such as Tim at Performive, if you are storing in their region (North America).

https://plus.fil.org/verifiers

**Comment by TimWilliams00** [comment]
I have no datacap left now.

**Comment by Jacintos** [comment]
@TimWilliams00 Thank you!
@galen-mcandrew I think we can not wait for your testing such a long time. If it can't be fixed, can you open a new window for us?

**Comment by galen-mcandrew** [comment]
@Jacintos There are other notaries in your region as well: https://plus.fil.org/verifiers

**Comment by galen-mcandrew** [comment]
Also seeing another LDN request by same user, for a very different project. Can you clarify your role and connection with these official organizations? Do you have an official email for this company, with the toastmasters domain?

**Comment by Jacintos** [comment]
Hi Galen, I'm Jacintos. I'm a data partner who apply datacap for Toastmaster. Like the Filecoin Gravity Project(https://protocollabs.typeform.com/to/r4INu5Ud?typeform-source=filecoin.io), my responsibility is to contact more data providers and help them know Filecoin clearly. And I will also help them find suited storage providers under the rules of community. Cause they are not familiar with the application process of github, I do the application on their behalf.

Toastmaster is a club which teaches public speaking and leadership skills with membership system. They often organize lectures so that they have amount of videos need to record and store. These datasets are public. This time they contact me to help them use filecoin to store.

You can refer their contents on youtube.
https://www.youtube.com/channel/UC4GYJFyJ7cZVfqMNF0JZfTQ

**Comment by Jacintos** [comment]
Hi Galen, I'm Jacintos. I'm a data partner who apply datacap for Toastmaster. Like the Filecoin Gravity Project(https://protocollabs.typeform.com/to/r4INu5Ud?typeform-source=filecoin.io), my responsibility is to contact more data providers and help them know Filecoin clearly. And I will also help them find suitable storage providers under the rules of community. Cause they are not familiar with the application process of github, I do the application on their behalf.

Toastmaster is a club which teaches public speaking and leadership skills with membership system. They often organize lectures so that they have amount of videos need to record and store. These datasets are public. This time they contact me to help them use filecoin to store.

You can refer their contents on youtube.
https://www.youtube.com/channel/UC4GYJFyJ7cZVfqMNF0JZfTQ

**Comment by IPFSUnion** [comment]
Happy to support.

**Comment by Jacintos** [comment]
@galen-mcandrew Hi, can you give us a new RKH or any access which can make my issue continue? It's been a long time that this issue stopped.

**Comment by Jacintos** [comment]
@galen-mcandrew Hi, can you give us a new Multisig Notary Reconnection Request to continue this issue? It's been a long time that this issue stopped.

**Comment by Jacintos** [comment]
@TimWilliams00 @XnMatrixSV @dkkapur @galen-mcandrew @IPFSUnion @dannyob @swatchliu @MasaakiNawatani @Reiers @MegTei Please help me about this issue! I have no way of how to continue it.

**Comment by Jacintos** [comment]
@TimWilliams00 @XnMatrixSV @dkkapur @galen-mcandrew @fabriziogianni7 @IPFSUnion @dannyob @swatchliu @MasaakiNawatani @Reiers @MegTei Please help me about this issue! I have no way of how to continue it.

**Comment by Jacintos** [comment]
@TimWilliams00 @XnMatrixSV @dkkapur @galen-mcandrew @fabriziogianni7 @IPFSUnion @dannyob @swatchliu @MasaakiNawatani @Reiers @MegTei Please help me about this issue! I have no way of how to continue it. Why is it so inefficient about handling?

**Comment by swatchliu** [comment]
> Since no one has supplied actual evidence of working with this official entity, we are reaching out to Toastmasters directly over email.
 That's a good  method to verify the relationship between the applicant and the company

**Comment by TimWilliams00** [comment]
Hi Galen @galen-mcandrew, I mentioned the previous contact between me and the chairman of Toastmaster (https://github.com/filecoin-project/filecoin-plus-large-datasets/issues/162#issuecomment-1004500009), and I invited them to experience the Filecoin network. In this case, I don't think a webmaster will konw any decision from management. Of course, I am not against your doing so.

**Comment by swatchliu** [comment]
I think there might some misunderstanding.  This is not disrespect for the efforts of notaries. All notaries are providing services to the community, all of them are deserve the greatest support and respect. @TimWilliams00 

Tim's logic is right, verifying with the webmaster is not the smart way. Not all decisions from the management team will be notified to every employee of this company, like the webmaster or marketing specialist.  @galen-mcandrew

Is there any smart way to verify the client?  Indeed, this application has taken too long time, and the working efficiency of RKH does need to speed up.

**Comment by MegTei** [comment]
> I think there might some misunderstanding. This is not disrespect for the efforts of notaries. All notaries are providing services to the community, all of them are deserve the greatest support and respect. @TimWilliams00
> 
> Tim's logic is right, verifying with the webmaster is not the smart way. Not all decisions from the management team will be notified to every employee of this company, like the webmaster or marketing specialist. @galen-mcandrew
> 
> Is there any smart way to verify the client? Indeed, this application has taken too long time, and the working efficiency of RKH does need to speed up.

Yes @swatchliu we should be doing client and project sponsor KYC as part of the application, not this far down the process. Doing it upfront will be more frictionless

**Comment by UnionLabs2020** [comment]
Anyway，this is a way we can have a try. If @TimWilliams00 has already done KYC of this issue, do the process need a second notary to perform double KYC ? If it's convenient, @Jacintos could you DM me the right person I can contact?

**Comment by galen-mcandrew** [comment]
Yes agree overall. If there is evidence of communication we would love to see it attached here (such as email screenshots). Happy to also reach out to the appropriate person directly, if you can provide that email address, for example if that's the chairman.

**Comment by UnionLabs2020** [comment]
<img width="718" alt="1" src="https://user-images.githubusercontent.com/68763845/154900285-940ab15c-816a-41b5-ad1e-dc8d37841f93.png">
<img width="720" alt="2" src="https://user-images.githubusercontent.com/68763845/154900304-00ebdefb-7000-40db-beff-861cb68ff065.png">
![a52c6dbd733638df105760df83b4893](https://user-images.githubusercontent.com/68763845/154900335-ee201d9a-129c-4114-b6a7-b58293760cb7.jpg)

**Comment by UnionLabs2020** [comment]
Confirmed with double KYC.  If the cooperation goes well later, I think we can invite him to promote filecoin in their club. Glad to support this application.

**Comment by Jacintos** [comment]
@galen-mcandrew Galen, now there are two notaries who proved for me. Can my issue continue again?

**Comment by Jacintos** [comment]
@TimWilliams00 @UnionLabs2020 Thank you for your help!
@galen-mcandrew Galen, now there are two notaries who proved for me. Can my issue continue again?

**Comment by UnionLabs2020** [comment]
It should go ahead after RKH passed.

**Comment by Jacintos** [comment]
@galen-mcandrew @dkkapur @fabriziogianni7 Hi, any update here?

**Comment by Jacintos** [comment]
@galen-mcandrew We have provided evidence already. So, what do you want me to provide for my issue? You can elaborate on it.

**Comment by flyworker** [comment]
I support @galen-mcandrew  for more KYC. wechat record is not enough. Anybody can change wechat name and make a record.

**Comment by dkkapur** [comment]
@Jacintos the ask here is for you to share the KYC evidence that you represent Toastmasters here for the community. If you can only share this privately with a small subset of notaries, please let the community know this as well so we can work on the right path forward for this (similar to https://github.com/filecoin-project/notary-governance/issues/489).

**Comment by Jacintos** [comment]
@dkkapur Yes, I think we need more detailed KYC rules in order to find an appropriate balance between KYC and customers' privacy. Welcome more notaries in Asia or some dedicated team to push this process that shouldn't be so hard and move forward on it.

**Comment by dkkapur** [comment]
@Jacintos that does not answer the question. The specific ask is - 
1. share the KYC evidence that you represent Toastmasters here OR
2. state that you cannot share this publicly, and identify >4 notaries that are willing to support you from 3 different regions, and show this to them privately, similar to what https://github.com/filecoin-project/notary-governance/issues/489 the client did here to build confidence in the community. 

Toastmasters is a massive global organization. Your GitHub account was created <1 year ago and has been used to file applications for two other projects as well. It is reasonable for you to establish why/how you are representing various organizations that are otherwise not related to each other.

1. https://github.com/filecoin-project/filecoin-plus-client-onboarding/issues/1060
2. https://github.com/filecoin-project/filecoin-plus-large-datasets/issues/187

**Comment by swatchliu** [comment]
We talked to Jacintos before. AFAIK, she is the data partner who applies on behalf of these organizations. Also we had a video talk with toastmaster's district director whom Jacintos shared with us ( can find his name on their official media：https://mp.weixin.qq.com/s/Y4NQys-A0VEN7h7omAz-yA) to confirm this application, the result is positive. but he expressed it's not appropriate to post his photo here. We need to respect them and welcome each of the data partners in our commuinty.  We support this issue if needed!

**Comment by Jacintos** [comment]
Hi Galen and Deep @galen-mcandrew @dkkapur ,  can we set up a online call to get rid of all doubts and misunderstanding?

**Comment by dkkapur** [comment]
@Jacintos - thanks for sharing more information and background! However - we still need a public verification to show your relationship with Toastmasters. I think a few ways we can do this:

- if Mr. Lee/Toastmasters can email filplus@fil.org and we can share this with the notaries
- if Toastmasters can come with you to a Notary Governance call and can share the plan with the community
- or share contact / plan with just a subset of Notaries and do the same as https://github.com/filecoin-project/notary-governance/issues/489

What do you think is best here?

**Comment by galen-mcandrew** [comment]
Any update here? Can you please respond to the above and provide more evidence?

**Comment by Jacintos** [comment]
Hi @dkkapur and @galen-mcandrew , Mr. Li is the Toastmaster officer in China. He does not have the email of the Toastmaster domain. The authenticity of his identity can be confirmed through the official WeChat account.

It has been more than 5 months since the application was submitted 22 December 2021. I contacted Mr. Li last week, it is regrettable that Toastmaster China has no longer wanted to continue the project.

Please close the issue. Thank you for your cooperation and support.
